rainbowup/scripts/prepare.sh

```diff
@@ -0,0 +1,37 @@
+CORE_SRC="~/.rainbowup/core"
+BRIDGE_SRC="~/.rainbowup/bridge"
+
+while : ; do
+  case "$1" in 
+    --nearcore_source)
+       [ -n "${CORE_SRC}" ] && usage
+       $CORE_SRC="$2"
+       shift 2 ;;
+    --source)
+       [ -n "${BRIDGE_SRC}" ] && usage
+       BRIDGE_SRC="$2"
+       shift 2 ;;
+    *)
+       break ;;
+  esac
+done
+
+# Compile source of nearcore
+cd $CORE_SRC && cargo build --package neard --bin neard
+echo "Compiled source of nearcore"
```

Comment:
I feel this and below echos good self documented, so these comments are not very necessary

Reply:
done